tests/runtime/ray/test_integration.py

```diff
@@ -926,3 +926,32 @@ def test_outside_workflow(self):
 
         # Then
         assert ids == (None, None, None)
+
+
+@pytest.mark.slow
+class TestDictReturnValue:
+    """
+    Tasks that had dicts in return statement used to cause WF failures.
+    This test might look trivial bul unless AST code was majorly refactored,
+    please abstain from removing it (also refer to git history for the fix itself)
+    """
+
+    def test_dict_as_task_return_value(self, runtime: _dag.RayRuntime):
+        @sdk.task(source_import=sdk.InlineImport())
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏
This is the default now.
```suggestion
        @sdk.task
```